src/library/blas/gens/clTemplates/trsv_gemv.cl

```diff
@@ -1420,7 +1420,8 @@ __kernel void %PREFIXtrsv_CLT_ComputeRectangle_kernel( 	__global const %TYPE* _A
 	int blkid = get_group_id(0);
 	int V= %V;
 
-	__local %TYPE solved[%TRIANGLE_HEIGHT];
+	__local %TYPE%V solvedV[%TRIANGLE_HEIGHT/%V];
+	__local %TYPE *solved = (__local %TYPE *)solvedV;
```

Comment:
The introduction of %V is the bugfix, and it mirrors a change already merged in another PR: basically if the `__local` type is not the vector type, but the scalar one, there is no guarantee that the array will be allocated with the kind of alignment needed for vector access. For example, this fails on NVIDIA GPUs. By declaring the array with the vector type, correct alignment is ensured, and both scalar and vector access can be achieved.